xml/day2_ui_documentation.xml

```diff
@@ -158,6 +158,86 @@
   </mediaobject>
  </figure>
 </section>
+<section xml:id="admin_ui_suse_cloud_configuration">
+ <title>Configuration</title>
+ <para>
+  The Configuration tab displays services that are deployed in the cloud and
+  the configuration files associated with those services. Services
+  may be reconfigured by editing the `.j2` files listed and clicking the
+  "Update" button.
+ </para>
+ <figure xml:id="admin_ui_cloud_service_configuration">
+  <title>&clm; Admin UI &suse; Service Configuration</title>
+  <mediaobject>
+   <imageobject role="fo">
+    <imagedata fileref="clm_admin_services_configuration_0.png" width="75%"/>
+   </imageobject>
+   <imageobject role="html">
+    <imagedata fileref="clm_admin_services_configuration_0.png"/>
+   </imageobject>
+  </mediaobject>
+ </figure>
+ <para>
+  Clicking one of the listed files will open up the file editor where
+  changes can be made to the service configuration. Files that have
+  been edited but have not had their updates applied to the cloud
+  will be identifed by an asterisk `*` in the file list.
+ </para>
+ <figure xml:id="admin_ui_cloud_service_configuration_editor">
+  <title>&clm; Admin UI &suse; Service Configuration Editor</title>
+  <mediaobject>
+   <imageobject role="fo">
+    <imagedata fileref="clm_admin_services_configuration_1.png" width="75%"/>
+   </imageobject>
+   <imageobject role="html">
+    <imagedata fileref="clm_admin_services_configuration_1.png"/>
+   </imageobject>
+  </mediaobject>
+ </figure>
+ <para>
+  After editing the service configuration, click the "Update" button
+  to begin deploying configuration changes to the cloud. The status of
+  those changes will be streamed to the UI.
+ </para>
+ <figure xml:id="admin_ui_cloud_service_configuration_update">
+  <title>&clm; Admin UI &suse; Service Configuration Update</title>
+  <mediaobject>
+   <imageobject role="fo">
+    <imagedata fileref="clm_admin_services_configuration_2.png" width="75%"/>
+   </imageobject>
+   <imageobject role="html">
+    <imagedata fileref="clm_admin_services_configuration_2.png"/>
+   </imageobject>
+  </mediaobject>
+ </figure>
+</section>
+<section xml:id="admin_ui_suse_cloud_model">
+ <title>Model</title>
+ <para>
+  The Model tab displays the yml files that make up the primary configuration
```

Comment:
The <guimenu>Model</guimenu> tab

Reply:
filename-tagModelclose-filename-tag

Reply:
I'm a little fuzzy on this one, are tabs usually wrapped as files?

Reply:
I see the other tab mentions as guimenu, I'll try using that, let me know if thats not right in this spot

Reply:
I was wrong. `guimenu` it is
files and directories are wrapped with `filename`